Assets/Scripts/Weapons/ProjectileGun/Bullet.cs

```diff
@@ -7,6 +7,11 @@
 
 public class Bullet : MonoBehaviour
 {
+    // Variables changeable by items
+    public static float C_Damage = 1f;
+    public static int C_BulletPierce = 1;
+    public static float C_Knockback = 1f;
+

```

Comment:
Can I ask why these variables are named the way that they are?

Reply:
Changeable_Damage -> C_Damage to differentiate the naming of WeaponData

Reply:
Since we can't edit WeaponData `C_` variables are what items can edit to effect gameplay.